examples/demo-rollup/src/test_rpc.rs

```diff
@@ -98,18 +96,34 @@ fn test_helper(data: String, expected: &str, port: u16) {
 
         let ledger_rpc_module = ledger_rpc::get_ledger_rpc::<i32, i32>(ledger_db.clone());
 
+        let actual_port_placeholder: Arc<AtomicU16> = Arc::new(AtomicU16::new(0));
```

Comment:
(non-blocking, question) With an atomic, just a `&` reference should be safe to share between threads. Does the compiler complain because it's not `'static`?